src/main/java/fizzbuzz/FizzBuzz.java

```diff
@@ -4,15 +4,29 @@
 package fizzbuzz;
 
 public class FizzBuzz {

```

Comment:
strictly speaking not Single Responsibility.
Class does the "calculation" and the output